ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/recovery/SuspendBlockingRecoveryModule.java

```diff
@@ -3,7 +3,7 @@
 /**
  * If a recovery module implements this interface it allows the recovery manager shutdown to block
  */
-public interface ShutdownBlockingRecoveryModule extends RecoveryModule {
+public interface SuspendBlockingRecoveryModule extends RecoveryModule {
```

Comment:
I note that XARecoveryModule needed to be changed to deal with this. I have a local change

Reply:
@jmfinelli I have raised https://github.com/jmfinelli/narayana/pull/2 against your branch for that one